lib/SILOptimizer/Differentiation/JVPCloner.cpp

```diff
@@ -920,7 +980,10 @@ class JVPCloner::Implementation final
     auto *mappedAllocStackInst = diffBuilder.createAllocStack(
         asi->getLoc(), getRemappedTangentType(asi->getElementType()),
         asi->getVarInfo());
-    bufferMap.try_emplace({asi->getParent(), asi}, mappedAllocStackInst);
+    auto insertion =
+        bufferMap.try_emplace({asi->getParent(), asi}, mappedAllocStackInst);
+    if (insertion.second)
```

Comment:
Should we instead `assert(insertion.second)`?

Reply:
Replaced all the `bufferMap.try_emplace` calls with `setTangentBuffer` (which is exactly `bufferMap.try_emplace` + `assert(inserted.second)`. It seems `bufferMap.try_emplace` uses were a bug.